aplus_client/django/models.py

```diff
@@ -22,7 +21,7 @@ def get_by_url(cls, url):
         hostname = urlsplit(url).hostname
         if not hostname:
             raise ValueError("Url doesn't have hostname")
-        obj, created = cls.objects.get_or_create(domain=hostname)
+        obj, created = cls.objects.get_or_create(domain=hostname) # pylint: disable=unused-variable
```

Comment:
Should we actually use prefix 'created' with _ ? [Apparently that would make linter happy](https://softwareengineering.stackexchange.com/questions/139582/which-style-to-use-for-unused-return-parameters-in-a-python-function-call). I guess we do not want to use plain underscore, to avoid confusion with gettext.